cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala

```diff
@@ -65,6 +65,17 @@ SHOW DATABASE myDatabase
 
 List information about the database `myDatabase`.
 
+###assertion=show-one
+//
+
+SHOW DATABASES
+YIELD name, currentStatus
+WHERE name CONTAINS 'my' AND currentStatus = 'online'
+###
+
+List information about databases, filtered by name and online status and further refined by conditions on these.
```

Comment:
Not sure what I think of the use of `filtering` here, I guess we're filtering the columns so it should be fine but my brain still connects it to filtering rows which is what your `refined by` does :P (this goes for all commands not just database)

Reply:
I think that for a reader of the manual, the wording will be fine as `filtering` is not per definition associated with `columns`.